frontend/HelperFuncs/utils.js

```diff
@@ -249,3 +249,48 @@ export const getMatchingUsers = async(searchTerm) => {
     console.log(data)
     return data;
 }
+
+
+/*
+helper function to format the rows of the AG-Grid component on search pages
+*/
+export function getSearchRows(userData) {
+    console.log('userData', userData)
```

Comment:
extra log

Reply:
this was deleted in the following pr on this branch